### DISCUSSION TITLE
CFAST: Error in &OUTP

### DISCUSSION THREAD

**Discussion by SigrinDeerns** [firemodels/cfast#2142] [Q&A]
I am doing a series of CFAST calculations to check influence of variating of some parameters on smokelayer. Everytime I am changing 1 extra parameter by making a new file by 'save as'. Most of the times this is going okay. But sometimes this results in an error, once I run the model again with 1 change, see below. I know CFAST doesn't like long paths so I have saved alle calculations in a map on de Destktop. Am I doing wrong something?

CFAST: CFAST exited with a non-zero exit code: 1
CFAST: 1

CFAST: Error in &OUTP: Too many &OUTP entries. Maximum is2525                                                                          
Error on line 2621 of the input file: &OUTP ID = 'Bedroom 3 Alarm'
***Error exit from CFAST, error 15 from routine read_dump
Error in &OUTP: Too many &OUTP entries. Maximum is2525                                                                          

Log: ***Error exit from CFAST, error 15 from routine read_dump
Log: Error on line 2621 of the input file: &OUTP ID = 'Bedroom 3 Alarm'
Log: 
Log: Compilation Date : Fri 05/13/2022  01:48 PM
Log: Revision Date    : Thu May 12 10:19:24 2022 -0400
Log: Revision         : CFAST7.7.3-4-g0c33733b
Log: Release Version  : CFAST 7.7.3
Log: 
Log: CFAST

Thanks for helping me out, Sigrin

**Comment by preneke** [comment]
Can you attach the input file? Also how many &OUTP namelist entries do you have? 

Thanks

**Comment by SigrinDeerns** [comment]
Hi,

The file was to big to send. So by wetransfer: https://we.tl/t-uwYJRRwV0n
I have multiple files for this one project, that all result in the same error.
But also files that do not have this error.

Only ‘save as’ in some cases lead to this error.
So I want to change the model (1 aspect), I first ‘ save as’  under a new name (1 change of scenario number for example). Then before changing the model I try to run the file. Sometimes then already only after saving under a new name, I have this error.

Met vriendelijke groet,

Sigrin Drost
Senior adviseur


[Deerns Logo]

T +31 6 50 16 34 04

***@***.******@***.***>
www.deerns.nl

Deerns Nederland B.V.
Westerlaan 111
8011 CA Zwolle

**Reply by rpeacoc** [reply]
CFAST has a default limit of 2500 OUTP inputs for an input file. The input file you sent has a single compartment with more than 100 000 OUTP inputs that seem to be for numerous compartments that are not in your simulation. You'll need to modify the outputs in your input file to match the actual simulation.
